static/index.html

```diff
@@ -39,12 +39,13 @@ <h1 class="title">Remote Play</h1>
       <button onclick="callEndpoint('/seek_right')" class="button forward">
         &gt;&gt;
       </button>
+      <div id ="wrapper">
       <div class="touchpad" id="touchpad">
-        <p>
-          Use this area as a touchpad.<br />
-          Supported gestures:<br />
-          Track, tap (Left Mouse Button), and double tap (Right Mouse Button).
-        </p>
+      </div>
+      <div class="vscrollpad" id="vscrollpad">
+      </div>
+    </div>
```

Comment:
nit, fix indentation in this area. Is it intentional for wrapper exclude scrollpads?

Reply:
The wrapper only excludes the horizontal scroll pad so that it will appear at the bottom of the touchpad itself.

Reply:
Got it, let's indent the contents inside wrapper to make it more readable?